pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go

```diff
@@ -0,0 +1,616 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import (
+	"fmt"
+	"reflect"
+	"sync"
+
+	v1 "k8s.io/api/core/v1"
+	quotav1 "k8s.io/apiserver/pkg/quota/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+)
+
+type GroupQuotaManager struct {
+	// Lock used for resourceKeys/quotaInfoMap/quotaTreeWrapper change
+	Lock                sync.RWMutex
+	totalResourceIndeed v1.ResourceList // totalResource without systemQuotaGroup and DefaultQuotaGroup's used Quota
```

Comment:
totalResourceIndeed is a little bit strange, I prefer offer a new function of GetTotalResourceForRuntimeQuotaCalculate() and substract system and default livly.